test/Truncate.js

```diff
@@ -497,6 +497,45 @@ describe('<Truncate />', () => {
             }
         });
 
+        describe('font', () => {
+            it('should use the native canvas font when no overrides are provided', () => {
+                const component = renderIntoDocument(
+                    <Truncate>
+                        abc
+                    </Truncate>
+                );
+
+                expect(component.canvasContext.font, 'to be', 'normal normal 10px sans-serif');
+            });
+
+            it('should use a single override when one is provided', () => {
```

Comment:
I like this assertion, but it will be hard to reliably test across implementations.

I'd drop it for now - testing no/all overrides seems good enough in this case